pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -258,7 +258,7 @@ func convertToVersion(obj runtime.Object, groupVersion unversioned.GroupVersion)
 // PortsExist checks if service has ports defined
 func (k *Kubernetes) PortsExist(name string, service kobject.ServiceConfig) bool {
 	if len(service.Port) == 0 {
-		logrus.Warningf("[%s] Service cannot be created because of missing port.", name)
+		logrus.Warningf("[%s] No ports defined, we will create a Headless service.", name)
```

Comment:
Does user even cares about this?
Maybe we can get rid of this message completely.